modules/system/utility-wfrp4e.js

```diff
@@ -1331,9 +1332,49 @@ export default class WFRP_Utility {
     return path.replace("tokens/popout/", "tokens/");
   }
 
-  static sleep(ms)
-  {
-    return new Promise(resolve => setTimeout(resolve, ms));
+  static async sleep(ms) {
+    await new Promise(resolve => setTimeout(resolve, ms));
+  }
+
+  static getActorOwner(actor) { 
+    if (actor.hasPlayerOwner) {
+      for (let u of game.users.contents.filter(u => u.active && !u.isGM && u.name != "Stream")) {
+        if (u.character?.id === actor.id) {
+          return u;
+        }
+      }
+      for (let u of game.users.contents.filter(u => u.active && !u.isGM && u.name != "Stream")) {
```

Comment:
Do we need a "Stream" check? Why would a stream be owner (instead of say, observer)?

Reply:
you are right, i can remove that.